### ISSUE TITLE
Change Directory Structure of Models

### ISSUE THREAD

**Issue by eugene-davis** [IndustrialDragonfly/DEdC#78]
All models should be in a common folder (so that common things - like the ID class and the BadConstructorException - can be in the top level of it for accessibility), with sub directories for each model.
Everyone will have needed to be working from the same common merge before this, so that it doesn't cause massive merge issues.
We should attempt to do this "properly" using Git merge.

**Comment by eugene-davis** [comment]
Did after code review.
